cvent-technology-radar.csv

```diff
@@ -132,6 +132,8 @@ Open API,Adopt,Tools,FALSE,"<p>Open API is how we define specifications for all
 Swagger,Hold,Tools,FALSE,<p>Swagger is an api definition format for REST services. Swagger is comparable to RAML and is really the first version of OpenAPI. We've consolidated our api definition language on Open API</p>
 webdriver.io,Adopt,Tools,FALSE,<p>webdriver.io is the test automation framework we're using for end to end tests. We prefer this framework because it supports selenium and chromium based workflows that allow for speed as well as comprehensive coverage across all the browsers that our products support</p>
 jest,Adopt,Tools,FALSE,<p>Jest is the unit testing tool we use for typescript/javascript based projects at Cvent. We also leverage jest for integration tests when it makes sense</p>
+enzyme,Hold,Tools,FALSE,<p>Enzyme is widely at Cvent for React unit testing, but it has been poorly maintained for several years and looks unlikely to ever support React >17</p>

```

Comment:
Given the html tags, should this be "React&gt;17"? I'm not sure how this radar stuff works exactly.

Reply:
Good call. Switched to words instead of > to be safe

Reply:
I actually type &amp;&gt;, but it turns out that github comments support html?